client/store/types.js

```diff
@@ -0,0 +1,5 @@
+const TYPES = {
+  SET_HEROES: 'SET_HEROES',
+};
+
+export default TYPES;
```

Comment:
You can do just
export const SET_HEROES = 'SET_HEROES';
export const SET_HEROE = 'SET_HEROE';

so you will not need to destructure them in reducer and action :)

Reply:
I like that you export TYPES and not many types.